lib/index.js

```diff
@@ -1,14 +1,19 @@
-const restify = require('restify');
-const bunyan = require('bunyan');
-const debug = require('debug')('bravado:server');
-const path = require('path');
-const packageJson = require('../package.json');
-const config = require('./config');
-const bravado = require('bravado-core');
-const cors = require('./cors');
-const routeHandler = require('./route');
-const indexRouteHandler = require('./index-route');
-const { forEachSerialP, keys, defer } = require('omnibelt');
+import restify from 'restify';
+import bunyan from 'bunyan';
+import _debug from 'debug';
+const debug = _debug('bravado:server');
+import path from 'path';
+import { readFileSync } from 'fs';
+import * as url from 'url';
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
+const packageJson = JSON.parse(readFileSync(path.resolve(__dirname, '../package.json')));
+import config from './config.js';
+import bravado from 'bravado-core';
+import cors from './cors.js';
+import routeHandler from './route.js';
+import indexRouteHandler from './index-route.js';
+import omnibelt from 'omnibelt';
+const { forEachSerialP, keys, defer } = omnibelt;
```

Comment:
in this file the main changes are exporting start and stop individually instead of as a default object. If we want to also do a default object I think that might make sense so I person can write 
```
import server from 'bravado-server' 

or

import { start, stop } from 'bravado-server';  
```

however the way it's written now if they wanted to do the first line they can do 

```
import * as server from 'bravado-server' 
```